.travis.yml

```diff
@@ -2,3 +2,5 @@ language: node_js
 
 node_js:
   - '6'
+  - '4'
+  - '0.10'
```

Comment:
Maintenance becomes a pain. If you aren't supporting it, I would have no reason to contribute back here, which would result in me working on a mostly separate project at that point (and re-publishing to NPM under a different name). If I'm working on my own fork, I'd probably just start changing the style/development practices to suit me also. The project would look a lot different after I played with it on my own for a week...